model/common/src/icon4py/model/common/decomposition/mpi_decomposition.py

```diff
@@ -15,7 +15,11 @@
 
 from gt4py.next import Dimension, Field
 
+from icon4py.model.common import dimension as dims
+from icon4py.model.common.config import Device
```

Comment:
this `Device`will go away (from the `model` code) so we cannot use it here.  See comment below